README.md

```diff
@@ -1,2 +1,55 @@
-# The-Next-Web
-A clone application of the next web original website
+
+# HTML/CSS capstone project
+
+This project is a responsive website application developed as a completion of HTML & CSS section of Microverse main curriculum. It is a collobtative project intended to showcase HTML & CSS Skills. It is a real-world-like project built with some business specifications. It is a clone of the thenextweb website. It consist of only one page, that is home page. It is adaptive to different standard screen size ( mobile , tablet and desktop). It is developed with HTML and CSS solo.
```

Comment:
This text contains some grammatical and typing errors, please fix them by inspecting the text with [Grammarly](https://app.grammarly.com/), you will be able to spot those errors

Reply:
We made some edits to the whole readme file, we hope to hear from you soon, Thank you

Reply:
Well done @prolajumokeoni , keep up the good work :muscle: :muscle: :muscle: